frontend/src/features/directory/components/FilterableVendorTable.tsx

```diff
@@ -34,7 +34,11 @@ function FilterableVendorTableContent({ uniqueRegions, tags, vendors, favoriteVe
   const searchQuery = searchParams.get(SEARCH_PARAM) || "";
   const travelsWorldwide = searchParams.get(TRAVEL_PARAM) === "true";
   const selectedSkill = searchParams.get(SKILL_PARAM) || "";
+  const lat = parseFloat(searchParams.get("lat") || "");
+  const lon = parseFloat(searchParams.get("lon") || "");
```

Comment:
i think you'll be doing this later, but it would be good for the user to be able to set the max distance as well!

Reply:
I expand radius based on results, but I don't let the user select it directly! mostly because it's unclear how radius should work if the user selects a larger region like state or country